router/batchrouter/batchrouter.go

```diff
@@ -2150,26 +2135,26 @@ func (brt *HandleT) holdFetchingJobs(parameterFilters []jobsdb.ParameterFilterT)
 }
 
 func IsAsyncDestination(destType string) bool {
-	return misc.Contains(asyncDestinations, destType)
+	return lo.Contains(asyncDestinations, destType)
```

Comment:
we could also use https://pkg.go.dev/golang.org/x/exp/slices#Contains
```suggestion
	return slices.Contains(asyncDestinations, destType)
```